commit r12-8527-g7811663964aa7e31c3939b859bbfa2e16919639f                                                                                                                             
Author: Martin Liska <mliska@suse.cz>                                                                                                                                                 
Date:   Wed Jun 29 15:28:07 2022 +0200                                                                                                                                                
                                                                                                                                                                                      
    libsanitizer: cherry-pick 791e0d1bc85d                                                                                                                                            
                                                                                                                                                                                      
    791e0d1bc85d: [compiler-rt] Add NO_EXEC_STACK_DIRECTIVE on s390x                                                                                                                  
    (cherry picked from commit aa87b7541b4c11f59c521154513f844ea6b5c977)                                                                                                              

--- libsanitizer/tsan/tsan_rtl_s390x.S
+++ libsanitizer/tsan/tsan_rtl_s390x.S
@@ -45,3 +45,5 @@ intercept setjmp, _ZN14__interception11real_setjmpE
 intercept _setjmp, _ZN14__interception12real__setjmpE
 intercept sigsetjmp, _ZN14__interception14real_sigsetjmpE
 intercept __sigsetjmp, _ZN14__interception16real___sigsetjmpE
+
+NO_EXEC_STACK_DIRECTIVE
